### PR TITLE
adding e-model-packager

### PR DESCRIPTION
**Pull request by AurelienJaquier**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,6 +119,7 @@ Useful links:
 **Runs cells from stand-alone packages**
 
 EModelRunner is a python library designed to run the cell models provided by the Blue Brain portals in a simple and straightforward way.
+The cells models that EModelRunner can run are created by [e-model-packager](https://github.com/BlueBrain/e-model-packager).
```
Comment (ilkilic) [inline README.md]: cells->cell ?

File: README.md [hunk 2]
```diff
@@ -157,3 +158,19 @@ Software repository to reproduce the results of the publication below.
 In the paper, we present a generalized automated workflow for the creation of robust electrical models and illustrate its performance by building cell models for the rat somatosensory cortex. Each model is based on a 3D morphological reconstruction and a set of ionic mechanisms. We use an evolutionary algorithm to optimize neuronal parameters to match the electrophysiological features extracted from experimental data. Then we validate the optimized models against additional stimuli and assess their generalizability on a population of similar morphologies. Compared to the state-of-the-art canonical models, our models show 5-fold improved generalizability. This versatile approach can be used to build robust models of any neuronal type.
 
 [GitHub repo](https://github.com/BlueBrain/SSCxEModelExamples)
+
+## e-model-packager
```
Comment (ilkilic) [inline README.md:163]: perhaps we can also add it to the bullet list at the top of the readme?

Comment (AurelienJaquier) [reply README.md:163]: good catch, done in latest commit

File: README.md (hunk 1 above)
Comment (AurelienJaquier) [reply README.md]: fixed in latest commit

**Review by ilkilic — APPROVED** [review]
Thank you!

**Pull request merged**